docs/adrs/0008-appliance-calculator-location.md

```diff
@@ -0,0 +1,83 @@
+- https://github.com/citizensadvice/content-api/tree/exp/interactive-tool-spike
+- https://github.com/citizensadvice/public-website/tree/exp/interactive-tool-spike
+
+## Decision
+
+1. Generalise from `energy-comparison-table` to `energy-apps`, by:
+   1. recreating the current app infrastructure with the more generic `energy-apps` name
+   2. switching to the the new sub-domain in production in the CloudFront config
+   3. renaming the `energy-comparison-table` app and namespacing the comparison table app appropriately
+2. Build the appliance calculator in the newly renamed `energy-apps` rails app
+
+## Consequences
+
+### Positive
+
+- all energy apps (excluding the smart meter tool) are contained in the same place
+- we will only need to solve certain problems once for all energy apps (eg monitoring, authentication, header and footer, country route handling)
+- little to no additional maintenance burden
+- can leverage the existing build and deployment workflows (which are robust and fast)
+- increased exposure to CDK and infrastructure configration for the energy team developer (the current infrastructure code was developed by the SRE team)
+
+
+### Negative
+
+- the additional work in re-creating and switching is significant, but this is ameliorated by the following factors:
+  - there is time to do this work now whilst the energy team wait for decisions about upcoming work (changes to the comparison table and the smart meter tool)
+  - it is easier to do now before more complexity is added to the single energy app
+- additional work in namespacing and organising the existing energy comparison table code
+
+## Discounted alternatives
+
+### Building the appliance calculator in public website
+
+The appliance calculator could also be built inside the public website codebase.
+
+#### Positives
+
+- problems like monitoring, authentication, header and footer, country route handling are all aready solved here
+- potentially easier / quicker for non-energy team devs to debug and support the appliance calculator code if required
+- little to no maintenance burdern
+
+#### Negatives
+
+- energy developers will need to understand the public website and content-api apps
+- long build times
+- poorer domain seperation - does this code really belong in the public website codebase?
+
+### Building the appliance calculator in a new, separate application and repository
+
+The appliance calculator is built completely seperately to everything else
+
+#### Positives
+
+- increased exposure to CDK and infrastructure configration for the energy team developer
+- best domain seperation
+
```

Comment:
Would this also allow for any non-energy interactive tools to be built easily in the future?